src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -376,7 +376,7 @@ const mapStateToProps = state => ({
   translate: translateWithDefaultMessage(getTranslate(state.localize)),
   currencyCode: state.session.currencyCode,
   allStatuses: state.purchaseOrder.statuses,
-  locale: state.session.activeLanguage,
+  locale: state.session.activeLanguageTag,
```

Comment:
what's the reason for that change? I mean:
1. why do we have to change it to the `activeLanguageTag`
2. if we even have to, why does it only take place on the PO list?

Reply:
I explained it in your first question (`Number.toLocaleString()` does not handle `es_MX` format)